libsoftwarecontainer/src/gateway/gateway.cpp

```diff
@@ -126,13 +126,13 @@ bool Gateway::isActivated()
     return m_state >= GatewayState::ACTIVATED;
 }
 
-ReturnCode Gateway::setEnvironmentVariable(const std::string &variable, const std::string &value)
+bool Gateway::setEnvironmentVariable(const std::string &variable, const std::string &value)
 {
     if (hasContainer()) {
-        return getContainer()->setEnvironmentVariable(variable, value);
+        return isSuccess(getContainer()->setEnvironmentVariable(variable, value));
```

Comment:
I guess `ContainerAbstractInterface::setEnvironmentVariable` should also be bool, eliminating the need for `isSuccess` here.